app/models/host/base.rb

```diff
@@ -90,11 +90,12 @@ def self.taxonomy_conditions
     # initializer and we set name when we are sure that we have primary interface
     # we can't create primary interface before calling super because args may contain nested
     # interface attributes
-    def initialize(*args)
+    def initialize(attributes = nil, &block)
```

Comment:
Looks like we don't override this in discovery: https://github.com/theforeman/foreman_discovery/blob/develop/app/models/host/discovered.rb